index.html

```diff
@@ -288,6 +288,24 @@ <h2>About me</h2>
             </li>
           </ul>
         </div>
+        <!-- contact form section -->
+        <div class="contact-section">
```

Comment:
- [ ] It would be great to wrap your contact form in a `section` that. This makes your webpage semantic and also improves your SEO for better visibility. [This](https://www.thoughtco.com/why-use-semantic-html-3468271) article provides an in-depth explanation of the importance of semantics HTML and its benefits.

Reply:
- Nice thanks 🤝